quantum-template.tex

```diff
@@ -101,7 +101,7 @@ \subsection{References and footnotes}
 Citations to other works should appear in the References section at the end of the work.
 
 \begin{theorem}[DOI links are required]
-  Important: As Quantum is a member of Crossref, all references to works that have a DOI must be hyperlinked according to the DOI. Those links must start with \texttt{https://doi.org/} (preferred), or \texttt{http://dx.doi.org/}. Direct links to the website of the publisher are not sufficient.
+  Important: As Quantum is a member of Crossref, all references to works that have a DOI must be hyperlinked according to the DOI. Those links must start with \texttt{https://doi.org/} (preferred), or \texttt{http://dx.doi.org/}. Direct links to the website of the publisher are not sufficient. Also so-called \href{https://shortdoi.org/}{shortDOIs} are unfortunately not accepted by Crossref (despite shotDOI being their own service). Thus all authors must use regular full DOIs that usually start with \mbox{``10.''} as in \url{https://doi.org/10.1103/PhysRevLett.120.103201} instead of the associated shortDOI starting with \mbox{``10/''} as in \url{https://dx.doi.org/10/gc5sj2}.
```

Comment:
```suggestion
  Important: As Quantum is a member of Crossref, all references to works that have a DOI must be hyperlinked according to the DOI. Those links must start with \texttt{https://doi.org/} (preferred), or \texttt{http://dx.doi.org/}. Direct links to the website of the publisher are not sufficient. Also so-called \href{https://shortdoi.org/}{shortDOIs} are unfortunately not accepted by Crossref (despite shortDOI being their own service). Thus all authors must use regular full DOIs that usually start with \mbox{``10.''} as in \url{https://doi.org/10.1103/PhysRevLett.120.103201} instead of the associated shortDOI starting with \mbox{``10/''} as in \url{https://dx.doi.org/10/gc5sj2}.
```